runtime/fundamentals/permissions.md

```diff
@@ -3,33 +3,39 @@ title: "Permissions"
 oldUrl:
   - /runtime/manual/basics/permissionsDeno/
   - /manual/basics/permissions
+  - /runtime/manual/basics/permissions
 ---
 
 Deno is secure by default. Therefore, unless you specifically enable it, a
-program run with Deno has no file, network, or environment access. Access to
-security sensitive functionality requires that permissions have been granted to
-an executing script through command line flags, or a runtime permission prompt.
-This is a major difference from Node, where dependencies are automatically
-granting full access to everything, introducing hidden vulnerabilities in your
-project.
+program run with Deno has no access to sensitive APIs, such as file system access, network connectivity, or environment access. You must explicitly grant access to these resources with command line flags or with the runtime permission prompt. This is a major difference from Node, where
+dependencies are automatically granted full access to everything, potentially
+introducing hidden vulnerabilities into your project.
 
-## Run untrusted code with confidence
+## Granting permissions
 
-Since Deno provides no I/O access by default, it's useful for running untrusted
-code and auditing third-party code. If you're building or extending a platform
-that runs user generated code, you can use Deno for running third-party code
-securely and host this code through
-[Deno Subhosting](https://deno.com/subhosting) or any other cloud platform of
-your choice.
-
-For the following example `mod.ts` has been granted read-only access to the file
-system. It cannot write to the file system, or perform any other security
-sensitive functions.
+To grant a permission to a script, you can use the `--allow-<PERMISSION>` flag
+when running the script. For example, to grant read access to the file system,
+you can use the `--allow-read` flag:
```

Comment:
You can also discuss the short versions of these flags, which are added in deno 1.46 - `-R` for `--allow-read` etc